src/components/Icon/BankIcons.ts

```diff
@@ -52,101 +14,85 @@ type BankIcon = {
     iconStyles?: Array<ViewStyle | CSSProperties>;
 };
 
+const BANK_NAMES = {
+    EXPENSIFY: 'expensify',
+    AMERICAN_EXPRESS: 'americanexpress',
+    BANK_OF_AMERICA: 'bank of america',
+    BB_T: 'bbt',
+    CAPITAL_ONE: 'capital one',
+    CHASE: 'chase',
+    CHARLES_SCHWAB: 'charles schwab',
+    CITIBANK: 'citibank',
+    CITIZENS_BANK: 'citizens bank',
+    DISCOVER: 'discover',
+    FIDELITY: 'fidelity',
+    GENERIC_BANK: 'generic bank',
+    HUNTINGTON_BANK: 'huntington bank',
+    NAVY_FEDERAL_CREDIT_UNION: 'navy federal credit union',
+    PNC: 'pnc',
+    REGIONS_BANK: 'regions bank',
+    SUNTRUST: 'suntrust',
+    TD_BANK: 'td bank',
+    US_BANK: 'us bank',
+    USAA: 'usaa',
+};
```

Comment:
I forgot to mention to always use `as const` for static objects like this, same thing with icon mappings

Reply:
👍🏻